src/test/shell/bazel/remote/remote_execution_test.sh

```diff
@@ -3232,44 +3232,73 @@ EOF
     --verbose_failures \
     //a:consumer >& $TEST_log || fail "Failed to populate the cache"
 
-  bazel clean || fail "Failed to clean"
+  bazel clean >& $TEST_log || fail "Failed to clean"
 
-  # download top level outputs without remote metadata
+  # download top level outputs
   bazel build \
     --remote_cache=grpc://localhost:${worker_port} \
     --remote_download_toplevel \
     --verbose_failures \
```

Comment:
supernit: `--verbose_failures` is unlikely needed/we could add that to the blazerc for the whole test if that aids debugging failures.

Reply:
Done.